src/Plugin/Field/FieldWidget/MediaLibraryEditWidget.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\entity_reference_override\Plugin\Field\FieldWidget;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\Component\Utility\NestedArray;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\media_library\Plugin\Field\FieldWidget\MediaLibraryWidget;
+
+/**
+ * Plugin implementation of the 'media_library_edit_widget' widget.
+ *
+ * @FieldWidget(
+ *   id = "media_library_edit_widget",
+ *   label = @Translation("Media library (with edit)"),
```

Comment:
We should have similar labels for both widget. Both should have either "(with override)" or "(with edit)".
I think, the former would be better.